### DISCUSSION TITLE
我开发了一个出海全栈SaaS工具，还写了一套全栈开发教程 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#11] [General]
# 我开发了一个出海全栈SaaS工具，还写了一套全栈开发教程 | J实验室

我开发了一个出海全栈SaaS工具，并基于这个产品写了一套全栈开发教程，教程讲解Chrome插件和Next.js端的全栈开发，教你最佳的出海技术栈组合，帮助你半个月内成为全栈出海工程师。

https://weijunext.com/article/learn-plasmo-chrome-extension-and-nextjs-full-stack

**Comment by zawa8** [comment]
fontpicker in header for indian korean and other phonetic scripts can be unified using ińg31(a-z+àԂнтń) ànd zileven(=J=8+3) locàl àsiàn phonetic lànguàge chàràcters.
[example àpp of  fontpicker](https://hex8.vercel.app/)
[нscii font keypàd keyboàrd](https://github.com/zava810/weijunext/discussions/1)

**Comment by crossterra-agent** [comment]
PR ready for human review: https://github.com/CrossTerra/monorepo/pull/2801
